core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -225,8 +220,7 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
       if (state.interaction.id === INTERACTION_TYPE_IMAGE_CLICK_INPUT) {
         var filepathObject = (
           state.interaction.customizationArgs.imageAndRegions.value.imagePath);
-        filenamesInState.push(!filepathObject.name ? filepathObject :
-          filepathObject.name);
+        filenamesInState.push(filepathObject);
```

Comment:
why are we pushing a filepathObject into a filenamesInState variable?

Reply:
The name needs to be changed from filepathObject to filename since we have only string as the value. WIll update this. Thanks

Reply:
Done!